### PR TITLE
Path names

### PR DESCRIPTION
**Pull request by milotictanja**
Path names have been corrected in all scripts.

### CODE REVIEW

**Comment by stijnvanhoey** [comment]
Can you run all scripts without errors?

File: src/p2a_analyses_read_processed_data.R [hunk 1]
```diff
@@ -27,7 +27,7 @@ data$yday <- as.numeric(data$yday)
 data$origin <- as.character(data$origin)
 
 # Gather Swedish data with all the same columns as df "data" created above
-source('sidescript_readSwedes.R')
+source("src/sidescript_readSwedes.R")
```
Comment (stijnvanhoey) [inline src/p2a_analyses_read_processed_data.R]: Please make sure to take into account the suggestion of @peterdesmet regarding the usage of the `here` package, see https://github.com/inbo/wmh-analysis/issues/1

Comment (stijnvanhoey) [reply src/p2a_analyses_read_processed_data.R]: same refers for all other changes to the relative paths

**Review by stijnvanhoey — CHANGES_REQUESTED** [review]
Good and important start! Make sure to test the successful output of the individual scripts.

File: src/p2a_analyses_read_processed_data.R (hunk 1 above)
Comment (milotictanja) [reply src/p2a_analyses_read_processed_data.R]: I made 2 further changes to the code: 
1. The `here` package is used for the relative paths now
2. I checked whether all scripts run without any errors and made some small corrections for that matter.

**Review by stijnvanhoey — APPROVED** [review]

**Comment by stijnvanhoey** [comment]
Thanks! Looks good to merge.

**Comment by peterdesmet** [comment]
Nice!

**Pull request merged**